app/controllers/idv/capture_doc_status_controller.rb

```diff
@@ -28,7 +28,7 @@ def status
           :unauthorized
         elsif document_capture_session.cancelled_at
           :gone
-        elsif rate_limiter.limited?
+        elsif rate_limiter.exceed_max?
```

Comment:
I am thinking whether we should completely remove this check, due to the way `rate_limiter` implemented.

And technically this `controller` wont be calle, since when `doc_auth` is submitted then the `rate_limit` should not be exceeded.

Reply:
I am trying out removing this call locally. Everything seems to be passing so far.

My main concern is that - if we shouldn't ever call this controller, is there some sort of earlier check that's misfiring? Because if it shouldn't have ever been called, then this user should never have encountered this issue...

Reply:
@night-jellyfish , i meant this wont be called technically if rate limited, unless something goes wrong or called intentionally.

Reply:
Can you point me to the earlier check that would filter out rate limited requests? I am having a hard time finding it.

Reply:
@night-jellyfish here, it's a `controller` `before_action` hook

https://github.com/18F/identity-idp/blob/add15b560cad78785afcbf87dd1634ef5cc046d3/app/controllers/idv/document_capture_controller.rb#L11-L11

Reply:
Thank you!! I also found [this relevant conversation](https://github.com/18F/identity-idp/pull/9370#discussion_r1357190769) that further supports your point.

With that in mind, I updated this PR to instead delete the check for rate limiting, as you suggested. I still need to test this manually.

Reply:
I think I need to also add a feature test for this, perhaps.